===================================================================
RCS file: /cvsroot/curl/curl/tests/data/test46,v
retrieving revision 1.13
retrieving revision 1.14
diff -u -r1.13 -r1.14
--- curl/tests/data/test46	2005/05/24 10:03:13	1.13
+++ curl/tests/data/test46	2006/02/06 20:02:14	1.14
@@ -43,7 +43,7 @@
 
 www.fake.come	FALSE	/	FALSE	1022144953	cookiecliente	si
 www.loser.com	FALSE	/	FALSE	1139150993	UID	99
-127.0.0.1	FALSE	/	FALSE	1139150993	mooo	indeed
+127.0.0.1	FALSE	/	FALSE	1439150993	mooo	indeed
 127.0.0.1	FALSE	/	FALSE	0	empty	
 </file>
 </client>
@@ -67,7 +67,7 @@
 
 www.fake.come	FALSE	/	FALSE	1022144953	cookiecliente	si
 www.loser.com	FALSE	/	FALSE	1139150993	UID	99
-127.0.0.1	FALSE	/	FALSE	1139150993	mooo	indeed
+127.0.0.1	FALSE	/	FALSE	1439150993	mooo	indeed
 127.0.0.1	FALSE	/	FALSE	0	empty	
 127.0.0.1	FALSE	/	FALSE	1262322000	ckyPersistent	permanent
 127.0.0.1	FALSE	/	FALSE	0	ckySession	temporary
